routes/base_node.js

```diff
@@ -56,4 +58,9 @@ router.get('/transactions', async (req, res) => {
   }
 })
 
+router.post('/proto', simpleAuth, async (req, res) => {
+  const result = await baseNode._checkVersion(req.query.update)
```

Comment:
I'm having a hard time passing the `update` param.

```bash
curl --data "update=1" http://localhost:4000/proto
```

returns the expected JSON, but `req.query.update` is `undefined`.

```bash
curl --data "update=1" http://localhost:4000/proto?update=1
```

returns `no matches found`.

However, I might be doing something wrong, since I'm not too good with `curl`. Any pointers?

Reply:
Never mind, I figured it out after GitHub syntax-highlighted the question mark. 🤦

Reply:
`curl -X POST http://localhost:4000/proto?update=1&token=token`

Reply:
Yeah, `zsh` requires a quoted URL. 🙄 😩